### PR TITLE
LRQA-66067 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
WYSIWYGUsecase#CanAddHyperlinkToExistingText
https://issues.liferay.com/browse/LRQA-66067

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase [hunk 1]
```diff
@@ -197,6 +197,34 @@ definition {
...
+		SelectFrame(value1 = "relative=top");
+
+		CKEditor.addEntryExternalURL(
+			entryExternalURL = "/web/guest/home",
+			fieldLabel = "content"
+		);
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CKEditor#BODY_FIELD_IFRAME_2");
+
+		AssertElementPresent(locator1 = "//a[normalize-space(text()) = 'Test Link' and contains(@href, '/web/guest/home')]");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase:225]: try to generalize into a poshi path file. If possible, avoid setting xpaths in .testcase or .macro level

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase [hunk 2]
```diff
@@ -197,6 +197,34 @@ definition {
...
+		SelectFrame.selectFrameNoLoading(locator1 = "CKEditor#BODY_FIELD_IFRAME_2");
+
+		SelectFieldText(locator1 = "CKEditor#BODY");
+
+		SelectFrame(value1 = "relative=top");
+
+		CKEditor.addEntryExternalURL(
+			entryExternalURL = "/web/guest/home",
+			fieldLabel = "content"
+		);
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase:221]: I'll move this into a new macro

**Pull request closed without merging**